parm/aero/variational/3dvar_gfs_aero.yaml

```diff
@@ -2,7 +2,9 @@ cost function:
   cost type: 3D-Var
   window begin: '$(AERO_WINDOW_BEGIN)'
   window length: $(AERO_WINDOW_LENGTH)
-  analysis variables: &3dvars [ua,va,t,ps,sphum,ice_wat,liq_wat,o3mr]
+  analysis variables: &3dvars [sulf,bc1,bc2,oc1,oc2,
```

Comment:
Still the question from before, does `sulf` work with the UFS model?

Reply:
Sounds like it should be `so4` according to Mariusz.

Reply:
Oh, I thought you were referring to the workflow. I'm running the ctests in this updated version now.

Reply:
Also, I noticed that Mariusz while changed the bumpparameters yamls, but not the staticb yaml's (which I will update), but they all passed the tests. But I'll sort this out in a while.